firebase-perf/src/main/java/com/google/firebase/perf/config/ConfigurationConstants.java

```diff
@@ -218,6 +230,11 @@ protected Long getDefault() {
       return 100L;
     }
 
+    @Override
+    protected Long getDefaultOnRcFetchFail() {
+      return getDefault() * 3;
```

Comment:
Let's add a comment to say that we are increasing the default 3 folds. Eg: 100ms to 300ms.

Reply:
Done